app/assets/src/components/common/CoverageVizBottomSidebar/CoverageVizBottomSidebar.jsx

```diff
@@ -434,7 +450,28 @@ export default class CoverageVizBottomSidebar extends React.Component {
         />
         <div className={cs.genomeVizRow}>
           <div className={cs.rowLabel}>Reference Accession</div>
-          <div className={cx(cs.genomeViz, cs.referenceAccession)} />
+          <BasicPopup
+            trigger={
+              <a
+                className={cx(cs.refAccessionVizLink, cs.genomeViz)}
+                href={`https://www.ncbi.nlm.nih.gov/nuccore/${
+                  currentAccessionSummary.id
+                }?report=genbank`}
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                <div
+                  className={cs.genomeVizInner}
+                  ref={refAccessionVizContainer => {
+                    this.refAccessionVizContainer = refAccessionVizContainer;
+                  }}
+                />
+              </a>
+            }
+            inverted
+            wide="very"
+            content={currentAccessionSummary.name}
```

Comment:
It might be nice to append a "click to open genback" or similar. It's not obvious what the effect of clicking should be.

Reply:
Will bring this up with Jenn.